src/main/java/org/example/Main.java

```diff
@@ -11,6 +11,7 @@ public static void main(String[] args) {
         Car car = new Car("Red", "ABCD", 105, 2000, "ABCD", 100, 50);
         Motorcycle motorcycle = new Motorcycle();
         Race race = new Race(vehiclesList, silverstone, 17, 2);
+        CargoTruck cargoTruck = new CargoTruck();
```

Comment:
we said that we're not going to make any modifcation on main.

Reply:
sorry